spec/workflow.builder.spec.ts

```diff
@@ -111,10 +111,9 @@ describe("workflow builder", () => {
 		
 		const workflowValidator = new WorkflowValidator({});
 		spyOn(workflowValidator, 'isValid')
-			.and.returnValue(false).and.callThrough();
```

Comment:
removed as it was invoking the real implementation https://jasmine.github.io/api/3.7/SpyStrategy.html